index.js

```diff
@@ -53,16 +53,27 @@ AFRAME.registerComponent("cubemap", {
       side: THREE.BackSide,
       transparent: true
     }).clone();
+    
+    //https://github.com/mrdoob/three.js/wiki/Migration-Guide#145--146
+    //they changed the name of the uniform from tCube to envMap, this variable helps us keep track of the name across three js versions
```

Comment:
Some minor style edits to this comment.

```suggestion
    // Starting in Three.js v146, `envMap` changed to `tCube`.
    // This variable helps us keep track of the name across Three.js versions.
    // https://github.com/mrdoob/three.js/wiki/Migration-Guide#145--146
```